package.json

```diff
@@ -38,11 +38,10 @@
     "debug": "^4.3.1",
     "napi-thread-safe-callback": "^0.0.6",
     "node-addon-api": "^4.2.0",
-    "node-gyp-build": "^4.3.0",
-    "@abandonware/bluetooth-hci-socket": "^0.5.3-8"
+    "node-gyp-build": "^4.3.0"
   },
   "optionalDependencies": {
-    "@abandonware/bluetooth-hci-socket": "^0.5.3-8"
+    "@abandonware/bluetooth-hci-socket": "^0.5.3-9"
```

Comment:
may i suggest to split this in 2 commits , i'd like have release changes minimal